Site_Files/src/main/webapp/script.js

```diff
@@ -27,6 +27,24 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+
+function getHistory() {
```

Comment:
Is this still used even though the things are no longer comments? If so, consider renaming to something else.